lib/services/oidc.go

```diff
@@ -102,8 +102,8 @@ func UnmarshalOIDCConnector(bytes []byte, opts ...MarshalOption) (types.OIDCConn
 		return nil, trace.Wrap(err)
 	}
 	switch h.Version {
-	case types.V2:
-		var c types.OIDCConnectorV2
+	case types.V2, types.V3:
```

Comment:
Perhaps this is standard practice elsewhere in the codebase, in which case ignore this comment, however if not then it may be confusing why we are using an `OIDCConnectorV3` for `case V2` and worthy of a clarifying comment.

Reply:
I have a comment in `(*OIDCConnectorV3).CheckAndSetDefaults()` already, but it's probably useful to restate it here as well.